### DISCUSSION TITLE
Can HFS Version 2.4.0RC6 Log Out Feature be used in Version 2.3m

### DISCUSSION THREAD

**Discussion by kenrmayfield** [rejetto/hfs#121] [Ideas]
Can HFS Version 2.4.0RC6 Log Out Feature be used in Version 2.3m? 

Thanks,
Kenneth

**Comment by rejetto** [comment]
hfs 2.3 doesn't support a real logout.
HFS 2.* is at https://github.com/rejetto/hfs2
while this repository is for HFS 3, which since it didn't officially replace HFS 2 yet has numbers like 0.*, to imply that it's not ready, but in my opinion is already way better than version 2.
I'm not personally working on 2 anymore.
